src/components/BtnNumbers.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes, jsx-quotes, react/prop-types */
```

Comment:
- [ ] Kindly make sure to update your readme, as requested by the previous reviewer.
- [ ] [OPTIONAL] Please refrain from using eslint disables as that defies the purpose of using linters in the first place.